build.sbt

```diff
@@ -10,6 +10,10 @@ lazy val docs = (project in file("docs")).
   settings(settings, publishArtifact := false).
   dependsOn(core)
 
+lazy val cats = (project in file("modules/cats")).
+  settings(settings, publishArtifact := false).
```

Comment:
Since we want to publish this module to Maven, `publishArtifact := false` should not be here.